paths/community_statistics.yaml

```diff
@@ -47,8 +47,6 @@ post:
                         type: string
                         format: bl-uuid
```

Comment:
you changed bl-uuid to uuid in a few places. uuids usually don't have the prefix, that's why it's bl-uuid in some places, because it's a custom extension on a normal uuid. 
but the most important part you probably it being consistent

Reply:
Ah, I thought that was just some inconsistency we put in there at some point. Will revert those.